masterlist.yaml

```
@@ -9515,6 +9515,36 @@ plugins:
       - crc: 0x7E676D2D
         util: 'TES4Edit v4.0.4'
 
+  - name: 'The Imperial Water( - BETTER CITIES)?\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/25804/'
+        name: 'The Imperial Waters'
+    msg:
+      - <<: *useOnlyOneX
+        subs: [ 'The Imperial Waters ESP' ]
+        condition: 'many("The Imperial Water( - BETTER CITIES)?\.esp")'
+  - name: 'The Imperial Water.esp'
+    msg:
+      - <<: *useInstead
+        subs: [ '[The Imperial Water - BETTER CITIES](https://www.nexusmods.com/oblivion/mods/25804/)' ]
+        condition: 'file("Better Cities Resources.esm") and not file("The Imperial Water - BETTER CITIES.esp")'
+    dirty:
+      - <<: *quickClean
+        crc: 0xDFCE7F93
+        util: '[TES4Edit v4.0.4](https://www.nexusmods.com/oblivion/mods/11536)'
+        itm: 19
+  - name: 'The Imperial Water - BETTER CITIES.esp'
+    msg:
+      - type: warn
+        <<: *patchUpdateAvailable
+        subs: [ '[Better Cities- The Imperial Waters Patch](https://www.nexusmods.com/oblivion/mods/48607/)' ]
+        condition: 'file("The Imperial Water - BETTER CITIES.esp") and not file("BC_IW_PATCH.esp")'
```

Should add a check for `Better Imperial City.esp` as well, since it's required by the patch.
```suggestion
        condition: 'file("The Imperial Water - BETTER CITIES.esp") and file("Better Imperial City.esp") and not file("BC_IW_PATCH.esp")'
```